IL/Internal/ILSession.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.Text;
+using MirrorSharp.IL.Advanced;
+using MirrorSharp.Internal.Abstraction;
+
+namespace MirrorSharp.IL.Internal {
+    // ReSharper disable once InconsistentNaming
+    internal class ILSession : ILanguageSessionInternal, IILSession {
+        private string _text;
+        private MirrorSharpILOptions _options;
+
+        public ILSession(string text, MirrorSharpILOptions options) {
+            _text = text;
+            _options = options;
+        }
+
+        public string GetText() => _text;
+
+        public void ReplaceText(string? newText, int start = 0, int? length = null) {
+            if (length > 0)
+                _text = _text.Remove(start, length.Value);
```

Comment:
I'll need to remember details here, but I think client sometimes sends text char-by-char, wonder if StringBuilder would be more efficient here.

Reply:
Probably! I've just copied code from FSharp version as an initial version.